applications/asset_tracker/src/main.c

```diff
@@ -1454,24 +1462,32 @@ void cloud_event_handler(const struct cloud_backend *const backend,
 		}
 
 #if defined(CONFIG_AGPS)
-		err = gps_process_agps_data(evt->data.msg.buf,
-					    evt->data.msg.len);
+#if defined(CONFIG_NRF_CLOUD_CELL_POS)
```

Comment:
Is this correct? Does `CONFIG_AGPS` have to be defined for cell pos support?

Reply:
No, that's for the AGPS helper lib. Fixed